lib/rule.js

```diff
@@ -21,7 +18,8 @@ Rule.prototype.match = function(message, slackClient) {
 };
 
 Rule.prototype.reactionMatches = function(message) {
-  return message.type === REACTION_ADDED && message.name === this.reactionName;
+  return (message.type === 'reaction_added' &&
+    message.name === this.reactionName);
```

Comment:
Can we add a constant somewhere, so that `reaction_added` isn't hard coded in both places? Would be great to also include a comment mentioning that it won't be needed once 2.0 is released, etc.